ISSUER_PROTOCOL.md

```diff
@@ -18,22 +18,29 @@ A Trust Token issuer should have an endpoint at a publicly accessible secure URL
 ```
 Key commitment result
 {
-  "protocol_version": <protocol version, TrustTokenV3PMB or TrustTokenV3VOPRF for this>,
-  "id": <key commitment identifier, as a monotonically increasing integer>
-  "batchsize": <batch size>,
-  "srrkey": <base-64 encoded SRRVerificationKey, in 32-byte RFC8032 encoding>,
-  <keyID>: { "Y": <base64-encoded TrustTokenPublicKey>,
-                     "expiry": <key expiry, encoded as a string representation of
-                                an integer timestamp in microseconds since the Unix
-                                epoch> },
-  <keyID>: { "Y": <base64-encoded public key>,
-                     "expiry": <key expiry, encoded as a string representation of
-                                an integer timestamp in microseconds since the Unix
-                                epoch> }, ...
+  <protocol_version>: {
+    "protocol_version": <protocol version, TrustTokenV3PMB or TrustTokenV3VOPRF for this>,
```

Comment:
Based on your comment below about being self-contained I assume we still want the protocol version in the key commitment object in addition to the dictionary key but just want to confirm

Reply:
Adding a sentence to clarify, but yes. Eventually we can maybe remove it, but it seems potentially useful to keep these self-contained/parseable without the top-level dictionary (if we want to change key commitment distribution or something).